documentation/docs/workflow.md

```diff
@@ -0,0 +1,149 @@
+
+Changelogs are generated for the "core" and every binding separately. To attach a PR to a specific changelog use the following label:
+
+##### `Rust` 
+Includes the PR in "core" / Rust changelog
+
+##### `Wasm`
+Includes the PR in Wasm changelog
+
+To list the PR title in a changelog the section to list in must be described through the following labels: 
+
+##### Changed
+Maps to the major version of [Semantic Versioning](#Semantic-Versioning).
+labels: `Breaking change`
+
+##### Added
+Maps to the minor version of [Semantic Versioning](#Semantic-Versioning).
+labels: `Added`
+
+##### Patch
+Maps to the patch version of [Semantic Versioning](#Semantic-Versioning).
+labels: `Patch`
+
+##### Deprecated
+Marks features that will be removed in the feature. No special version consideration apply here, since the feature did not change yet.
+labels: `Deprecated`
+
+##### Removed
+Marks features as being removed. Typically the features should have been deprecated in the previous version. This maps to the major version of [Semantic Versioning](#Semantic-Versioning). 
+labels: `Removed`
+
+##### Excluded Tags
+Marks changes that should not be part of the changelog. This should only be used for documentation and rare exceptions
+labels: `Documentation`, `No changelog`
+
+Please note that a PR can only be listed in one section of a changelog. So attaching the labels `Rust` `Added` `Patch` to a PR is invalid.
+
+##### Release Summary
+To attach a release summary to a version in the changelog create an issues with the label `release-summary`. Create a GitHub milestone matching the version you want to describe and attach it to the issue. The issue can be closed immediately. The text of the issue will be included in the changelog as the release summary.
+
+### Issue Labels
+The following labels are used to categorize issues, they don't have any effect on changelogs:`Request`, `Enhancement`, `Bug`, `Chore`, `Dependencies`, `Help wanted`, `Duplicate`, `release-summary`, `Wontfix`.
+
+## Release
+
+With the release process we can deliver versions of our software to the community. We use sensible automation where it helps to remove tedium. Some steps and decision remain manual, since they require active decision making.
+
+The final list of changes from the [changelog](#Changelog) informs the version of the release. If at least one change mapping to a major version is included, the major version needs to be incremented. In this case the minor and patch version are not incremented but set to `0`. If there are no change related to a major version, but changes related to a minor version are present the minor version needs to be incremented. The major version stays untouched, the patch version is set to `0` and so on. Determining the version of the release is the responsibility of the person doing the release.
```

Comment:
This kind of implies we need to generate the changelog before choosing the version, but we need to choose a version to generate a changelog.

```suggestion
The final list of changes from the [changelog](#Changelog) informs the version of the release. If at least one change mapping to a major version is included, the major version needs to be incremented. In that case the minor and patch version are set to `0`. If there are no change related to a major version, but changes related to a minor version are present the minor version needs to be incremented while the patch version is set to `0`. Otherwise only the patch version is incremented. Determining the version of the release is the responsibility of the person performing the release.
```